src/helpers/FontCollector.ts

```diff
@@ -0,0 +1,20 @@
+type Observer = (font: string) => void;
+
+export default class FontCollector {
+  private _fonts: string[] = [];
```

Comment:
```suggestion
  public readonly fonts: string[] = [];
```

That would remove the getter and having to name the variable `_`